tinyagentos/scope.py

```
@@ -0,0 +1,59 @@
+        async def wrapper(request: Request, *args, **kwargs):
+            scope = getattr(request.state, "token_scope", None)
+            if scope is None:
+                # No bearer token in play — session-cookie / other auth handles it.
+                return await fn(request, *args, **kwargs)
+            if not scope_matches(scope, action):
+                return error_response(
+                    status_code=403,
+                    error="scope_denied",
+                    detail=f"Token scope does not cover {action!r}.",
+                    fix=(
+                        f"Reissue the token with a wider scope (e.g. ['*'] for full "
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)